### ISSUE TITLE
Failed to execute menu action.

### ISSUE THREAD

**Issue by YounesSamatta** [azuravian/playnite-animatedgiffixer#2]
![Y7zftv3oY2](https://github.com/azuravian/playnite-animatedgiffixer/assets/96547485/9fcfb1c3-ffad-4b27-977e-23a40295fdf5)

**Comment by KiimaRo** [comment]
Yup, still an issue. Hopefully gets fixed soon.

**Comment by azuravian** [comment]
Sorry, I've been crazy busy with my new job.  This sounds like an easy fix though.  I'll try to take care of this soon.

**Comment by azuravian** [comment]
Check if the latest version works for you.  If not, let me know what error you get.

**Comment by YounesSamatta** [comment]
I got a slightly different error message this time using the new version  :
![Playnite DesktopApp_KBm1zzo7m6](https://github.com/user-attachments/assets/f5f5c9aa-d204-45e3-9c44-f1f646a20939)

**Comment by azuravian** [comment]
Ok, can you let me know which version of ImageMagick you're using.  It's obviously 7, but it must be a different version.

If you open a terminal or command prompt window and type "magick -version" you should get it.

**Comment by YounesSamatta** [comment]
Im using ImageMagick 7.1.1-36 Q16-HDRI x64

**Comment by azuravian** [comment]
OK, I've modified 3 locations where the convert submodule was used.  I just tested it and it worked for me.  Let me know how 1.3.1 does for you.

**Comment by Ryunoshi-source** [comment]
Yep, It works perfectly now, thanks for taking the time to fix it

**Comment by azuravian** [comment]
Closed with c6e1bb4
